lib/caching-browserify.js

```diff
@@ -95,7 +95,8 @@ module.exports = CoreObject.extend({
       packageCache: {},
       fullPaths: this.fullPaths,
       basedir: this._inputStaging,
-      debug: this.enableSourcemap
+      debug: this.enableSourcemap,
+      browserField: !process.env.EMBER_CLI_FASTBOOT
```

Comment:
Sorry, not sure I understand...which mode isn't a thing anymore?

Sorry I haven't had a chance to get back to tracking down the original error I was seeing when I first implemented this, we pushed off our fastboot deploy :/